shell/src/main/java/alluxio/cli/fs/command/LsPathInAlluxioCommand.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.file.URIStatus;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.ListStatusPOptions;
+import alluxio.util.FormatUtils;
+import alluxio.wire.BlockLocation;
+import alluxio.wire.FileBlockInfo;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+
+import javax.annotation.Nullable;
+
+/**
+ * Displays information for In-Alluxio data size under the path, grouped by worker.
+ */
+public class LsPathInAlluxioCommand extends AbstractFileSystemCommand {
```

Comment:
how is this different from `DuCommand`? can we add an option to `Du` to group output by workers?